includes/ucf-social-common.php

```diff
@@ -124,7 +123,90 @@ function ucf_social_icons_display_after( $atts ) {
 		}
 
 		/**
-		* Enqueue css for the social icons
+		* Displays the social social links
+		* @author RJ Bruneel
+		* @since 1.0
+		* @param array $atts | Assoc. array of shortcode options
+		* @return string
+		**/
+		public static function display_social_links( $atts ) {
+			$before = self::ucf_social_links_display_before( $atts );
+			if ( has_filter( 'ucf_social_display_before' ) ) {
+				$before = apply_filters( 'ucf_social_display_before', $before, $atts );
+			}
+
+			$content = self::ucf_social_links_display( $atts );
+			if ( has_filter( 'ucf_social_display' ) ) {
+				$content = apply_filters( 'ucf_social_display', $content, $atts );
+			}
+
+			$after = self::ucf_social_links_display_after( $atts );
+			if ( has_filter( 'ucf_social_display_after' ) ) {
+				$after = apply_filters( 'ucf_social_display_after', $after, $atts );
+			}
+
+			return $before . $content . $after;
+		}
+
+		/**
+		* Display the content before the social links
+		* @author RJ Bruneel
+		* @since 1.0
+		* @return string
+		**/
+		function ucf_social_links_display_before( $atts ) {
+			ob_start();
+		?>
+			<aside class="ucf-social-links">
+		<?php
+			echo ob_get_clean();
+		}
+
+		/**
+		* Display the social links content
+		* @author RJ Bruneel
+		* @since 1.0
+		* @param array $atts | Assoc. array of shortcode options
+		* @return string
+		**/
+		function ucf_social_links_display( $atts ) {
+			$atts = shortcode_atts( array(
+				'size'  => 'md'
+			), $atts );
+
+			$permalink = urlencode( get_permalink( $_POST->ID ) );
+
+			ob_start();
+		?>
+			<a class="btn btn-facebook color btn-<?php echo $atts['size']; ?>" target="_blank" href="https://www.facebook.com/sharer.php?u=<?php echo $permalink; ?>" title="Like this story on Facebook">
+				<span class="fa fa-facebook"></span> Like
+			</a>
+			<a class="btn btn-twitter color btn-<?php echo $atts['size']; ?>" target="_blank" href="https://twitter.com/intent/tweet?text=<?php echo get_the_title(); ?>&amp;url=<?php echo $permalink; ?>" title="Tweet this story">
```

Comment:
Something I know we haven't done in previous iterations of social buttons, but that we probably should do--when passing the post title in the twitter share url, we should probably urlencode it to prevent double-quotes within the post title from breaking the link, among other things.